.github/workflows/trigger

```diff
@@ -0,0 +1,10 @@
+VERSION=1.9.0-rc0
+
+pushd ~/go/src/github.com/kata-containers/packaging
+git tag --delete $VERSION
+git push archana :$VERSION
+git tag -a $VERSION -m "test tag - $VERSION" && git push archana $VERSION
+popd
+git tag --delete $VERSION
+git push egernst :$VERSION
```

Comment:
Why the `egernst` and `archana` in this file?

Reply:
This isn't intended to be merged.  Just a helper reference as the development phase finishes up. This'll be needed until the packaging bits land.